src/deployment/deploy.py

```diff
@@ -95,12 +94,7 @@
 )
 
 DOTNET_APPLICATION_SUFFIX = "-net"
```

Comment:
we should get rid of this too. and everything that references it.
This should take care of the extra -net registrations

Reply:
For this PR we stil need this. We are still going to be deploying two app functions until our instance has completely stopped using the -net endpoints from the agent side.